ios/FluentUI/Vnext/Popup Menu/PopupMenuItemCell.swift

```diff
@@ -27,6 +28,8 @@ public class PopupMenuItemCellState: MSFListCellState {
 			}
 		}
 	}
+	
+	// Disable selection for cell
```

Comment:
Is the intention for the comment to be public documentation? If so, we need to elaborate on what the disabled state entails and use the /// for the comment too.